### ISSUE TITLE
On Linux Debian Sid 32-bits : Stendhal does not display properly its game window

### ISSUE THREAD

**Issue by goupildb** [arianne/stendhal#174]
On Linux Debian Sid 32-bits, Stendhal does not display properly its game window:
![scr-0303](https://user-images.githubusercontent.com/7254537/41822064-494c25c6-77ea-11e8-9146-0ad8c5110e2b.png)

Console messages:
[Console.txt](https://github.com/arianne/stendhal/files/2131067/Console.txt)

**Comment by balaur64** [comment]
There seems to be an issue with multiple monitors, on newer Java versions.

**Comment by goupildb** [comment]
Thanks for the information

**Comment by AntumDeluge** [comment]
I have had a similar issue in the past (with multiple different versions of Java). But it seemed to only happen randomly once in a while. I would log in, here the startup music, but the screen would remain blank. Closing it & restarting the client usually worked. This issue occured on Windows & Linux (if I remember correctly) systems. If it is happening for you every time, it may be a different issue. There may be a VM argument to change graphics settings. I will see if can find something. What version of Java/OpenJDK are you using?

**Comment by kiheru** [comment]
This sounds like an experimental version of java, if I'm reading it correctly at all:

INFO  [main] stendhal.java                 ( 200) - Java-Runtime: OpenJDK Runtime Environment 11+18-Debian-1 from /usr/lib/jvm/java-11-openjdk-i386
INFO  [main] stendhal.java                 ( 203) - Java-VM: Oracle Corporation OpenJDK Server VM 11+18-Debian-1

(also the error pointed by the stack trace does not make much sense unless the bug is in the java libraries instead of stendhal)

**Comment by nhnb** [comment]
Java bug report: [JDK-8204646: IllegalArgumentException: Window must not be zero ](https://bugs.openjdk.java.net/browse/JDK-8204646)

**Comment by AntumDeluge** [comment]
Sounds like you should switch to Java 8 @goupildb.

**Comment by goupildb** [comment]
Yes it works with Java 8:
![capture du 2018-06-28 20-52-09](https://user-images.githubusercontent.com/7254537/42054644-3c1ef67a-7b15-11e8-9102-f52064d13f3f.png)
You were right, the problem comes from Java!
I also tested version 9, it does not work either.
Sorry I did not respond faster, I did not see your message.

So far the sound did not work on the game, but now it works well, I hear now the pretty soundtrack.

Thank you a lot for your answer and for your very good game.
Solved
